plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -159,6 +159,67 @@ protected TestTable createTableWithUnsupportedColumn()
                 "(one bigint, two decimal(50,0), three varchar(10))");
     }
 
+    @Test
+    public void testTimestampCoercionShowCreateTable()
+    {
```

Comment:
Is it possible to extract as a parameterized test ? Where we could pass the timestamp literal and expected timestamp type.

Reply:
Can we use `BaseConnectorTest#getColumnType` to verify the column types ?